### DISCUSSION TITLE
Seeking References for PelePhysics/Mechanisms/dodecane_lu

### DISCUSSION THREAD

**Discussion by RachelMeng0096** [AMReX-Combustion/PelePhysics#469] [Q&A] [answered]
Good evening everyone, where can I find the reference literature for the PelePhysics/Mechanisms/dodecane_lu mechanism?

**Comment by malihass** [comment, marked as answer]
Hi, the reference to cite is this one. Our mechanism is the same except it does not contain Argon.

@article{Yao2017,
  doi = {10.1016/j.fuel.2016.11.083},
  url = {https://doi.org/10.1016/j.fuel.2016.11.083},
  year = {2017},
  month = mar,
  publisher = {Elsevier {BV}},
  volume = {191},
  pages = {339--349},
  author = {Tong Yao and Yuanjiang Pei and Bei-Jing Zhong and Sibendu Som and Tianfeng Lu and Kai Hong Luo},
  title = {A compact skeletal mechanism for n-dodecane with optimized semi-global low-temperature chemistry for diesel engine simulations},
  journal = {Fuel}
}
